search/src/test/scala/weco/api/search/fixtures/ApiFixture.scala

```diff
@@ -83,20 +84,51 @@ trait ApiFixture extends AnyFunSpec with ScalatestRouteTest with IndexFixtures {
     routes: Route,
     path: String,
     unordered: Boolean = false
-  )(expectedResponse: (StatusCode, String)) =
-    eventually {
-      expectedResponse match {
-        case (expectedStatus, expectedJson) =>
+  )(expectedResponse: (StatusCode, String)): Assertion =
+    expectedResponse match {
+      case (expectedStatus, expectedJson) =>
+        def responseJson: Json = eventually {
           Get(path) ~> routes ~> check {
             contentType shouldEqual ContentTypes.`application/json`
-            parseJson(responseAs[String], unordered) shouldEqual parseJson(
-              expectedJson,
-              unordered
-            )
             status shouldEqual expectedStatus
+            parseJson(responseAs[String], unordered)
           }
+        }
+        responseJson shouldEqual parseJson(expectedJson, unordered)
+    }
+
+  def assertJsonResponseLike(
```

Comment:
These two new assertions are part of removing the fragility, but also make it easier to read the tests to understand what they are looking for.

Previously, the tests would all make a request, then check whether that whole response matched the content of the stored json file.

Unfortunately, this means that when we regenerate the test documents, we then had to modify the  expectations accordingly.

With this change, we can simply inspect the relevant parts of the response document.